indra/newview/llvoavatar.cpp

```diff
@@ -8234,22 +8393,43 @@ void LLVOAvatar::logMetricsTimerRecord(const std::string& phase_name, F32 elapse
 
 // call periodically to keep isFullyLoaded up to date.
 // returns true if the value has changed.
-BOOL LLVOAvatar::updateIsFullyLoaded()
+bool LLVOAvatar::updateIsFullyLoaded()
 {
-    ERezzedStatus rez_status = getRezzedStatus();
-    bool loading = getIsCloud();
+    S32 rez_status = getRezzedStatus();
+    bool loading = rez_status == 0;
     if (mFirstFullyVisible && !mIsControlAvatar)
     {
-        loading = ((rez_status < AV_REZZED_TEXTURED)
+        loading = ((rez_status < 2)
                    // Wait at least 60s for unfinished textures to finish on first load,
                    // don't wait forever, it might fail. Even if it will eventually load by
                    // itself and update mLoadedCallbackTextures (or fail and clean the list),
                    // avatars are more time-sensitive than textures and can't wait that long.
                    || (mLoadedCallbackTextures < mCallbackTextureList.size() && mLastTexCallbackAddedTime.getElapsedTimeF32() < MAX_TEXTURE_WAIT_TIME_SEC)
                    || !mPendingAttachment.empty()
-                   || (rez_status < AV_REZZED_FULL && !isFullyBaked())
+                   || (rez_status < 4 && !isFullyBaked())
```

Comment:
Is this an intentional change from 3 to 4 based on the changes previously made in maint-c?

Reply:
Yes, "fully rezzed" is 4 in `develop`.